zoo-css-challenge/style.css

```
@@ -1,195 +1,248 @@
+    font-size: 125%;
 }
 
 .image-container img {
-  vertical-align: top;
-  border: 5px solid #747c92;
+    vertical-align: top;
+    border: 5px solid #747c92;
 }
 
 .image-container img:first-of-type {
-  vertical-align: bottom;
+    vertical-align: top;
+    border-color: #56e39f;
+}
+
+.image-container img:last-of-type {
+    border-color: #ff3f00
 }
 
 .tigers {
-  background-color: #0b3c49;
-  color: #f0e100;
+    background-color: #45490bef;
+    color: #f0e100;
 }
 
 .giraffe {
-  background-color: #465775;
-  color: #56e39f;
+    background-color: #483C46;
+    color: #56e39f;
 }
 
 .giraffe h2 {
-  color: #ef6f6c;
+    color: #F4743B;
+}
+
+.giraffe p {
+    color: #BEEE62;
+}
+
+.giraffe p:first-child {
+    font-size: 20px;
+}
+
+.news h2 {
+    text-align: center;
 }
 
 .card {
-  display: inline-block;
-  background-color: white;
-  padding: 20px;
-  margin: 10px;
-  width: 30%;
-  border: 1px solid black;
-  border-radius: 10px;
+    display: inline-block;
+    background-color: white;
+    padding: 20px;
+    margin: 10px;
+    width: 30%;
+    border: 1px solid black;
+    border-radius: 10px;
 }
 
 .card h4 {
-  margin-bottom: 1em;
+    margin-bottom: 1em;
 }
 
 .card a {
-  text-decoration: underline;
-  font-weight: bold;
+    text-decoration: underline;
+    font-weight: bold;
 }
 
 .card time {
-  opacity: 0.5;
-  display: inline-block;
-  margin-bottom: 10px;
+    opacity: 0.5;
+    display: inline-block;
+    margin-bottom: 10px;
 }
 
+
 /*Education*/
 
+button:hover {
+    color: green;
+}
+
 .education {
-  height: 500px;
-  background-image: url("https://source.unsplash.com/1200x900/?tiger");
-  background-repeat: no-repeat;
-  background-size: cover;
-  background-position: center;
-  padding: 100px 5%;
+    height: 500px;
+    background-image: url("https://source.unsplash.com/1200x900/?tiger");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    padding: 100px 5%;
 }
 
 .education-container {
-  background-color: rgba(255, 255, 255, 0.75);
-  padding: 50px;
-  max-width: 400px;
+    background-color: rgba(255, 255, 255, 0.75);
+    padding: 50px;
+    max-width: 400px;
 }
 
 .education-container h3,
 .education-container p {
-  margin-bottom: 1em;
+    margin-bottom: 1em;
+    border-radius: 5px;
+}
+
+.education-container {
+    border-radius: 10px;
+}
+
+button:hover,
+button:focus {
+    background-color: #F4743B;
 }
 
+
 /*Membership*/
 
+.membership p:first-child {
+    width: 700px;
+    margin: 1rem 1rem 0.5rem 1.5rem;
+}
+
 .membership {
-  text-align: center;
-  background-color: #fffffc;
+    text-align: center;
+    background-color: #fffffc;
 }
 
 .membership h2 {
-  color: #ff3f00;
+    color: #ff3f00;
 }
 
 .membership strong {
-  color: #ff7f11;
+    color: #ff7f11;
 }
 
+
 /*Programs*/
 
 .programs .card {
-  display: block;
-}
+    display: block;
+    color: blue;
+}
+
+.programs {
+    width: 97%;
+    height: auto;
+    padding: 10px;
```

```suggestion
    padding: 0.75rem;
```
We should use relative units where possible, so `rem` instead of `px`